app/api.py

```diff
@@ -1,12 +1,22 @@
-
 from app.models import *
 from app import db
 
 import json
 import urllib
 import requests
+from marshmallow import Schema, fields
+
+
+# Schema For API
+class CarSchema(Schema):

```

Comment:
create a separate file for schemas, and import it from there.

Reply:
got it and done.